tests/package.lock

```diff
@@ -1,5 +1,19 @@
+sdk: ^2.0.0-alpha.64
 prefixes:
   cli: ..
+  host: pkg-host
 packages:
   ..:
     path: ..
+    prefixes:
```

Comment:
Why do we get host and fs through a local path?

Reply:
The fs package wasn't published yet.